src/libs/brand-utils.js

```diff
@@ -26,3 +43,14 @@ export const getEnabledBrand = () => Utils.cond(
   [isTerra(), () => brands.terra],
   () => brands.terra
 )
+
+export const pickBrandLogo = (color = false) => Utils.cond(
+  [isFirecloud(), () => color ? fcLogo : fcLogoWhite],
```

Comment:
What I meant is that each branch should return its own logo. I think you should be able to define everything about a brand in a single place.

Reply:
I think that would mean that each brand would need to return 2 different logos (color and white).

Reply:
Ah I see. Changed in [6ffd0e3](https://github.com/DataBiosphere/terra-ui/pull/3257/commits/6ffd0e33c6fe793ccda024345afc45e42bbbeb9c).

For the time being, I've put the references to logo files inside `brands`. Depending on how the `non-software engineer` persona shapes up, it may make sense to move the references to another place as they could be considered code changes rather than "config" changes.